app/assets/stylesheets/custom.scss

```diff
@@ -414,3 +449,255 @@ header {
     }
   }
 }
+
+input:focus {
+  outline: 0;
+}
+
+body {
+  background: $gray;
+}
+
+// 11. Tables (layout.scss)
+// ----------
+
+table {
+
+  tr {
+
+    &:nth-child(odd) {
+      background: #f9f9f9;
+    }
+
+    &:nth-child(even) {
+      background: #fff4f4;
```

Comment:
Because it is the one defined in the mockups 😅 
![Captura de pantalla 2020-12-10 a las 16 30 36](https://user-images.githubusercontent.com/16189/101792500-0f26ac00-3b05-11eb-8dcf-2adf7e9a70a6.png)

Reply:
:+1:

Since we've got about 12 usages of the `#fff4f4` color in the `custom.scss` file, how about extracting it to a variable?